cmd/kubeadm/app/cmd/phases/preflight.go

```diff
@@ -22,15 +22,37 @@ import (
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
 	"k8s.io/kubernetes/cmd/kubeadm/app/preflight"
+	kubeadmutil "k8s.io/kubernetes/cmd/kubeadm/app/util"
+	"k8s.io/kubernetes/pkg/util/normalizer"
 	utilsexec "k8s.io/utils/exec"
 )
 
+var (
+	masterPreflightLongDesc = normalizer.LongDesc(`
+		Run master pre-flight checks, functionally equivalent to what implemented by kubeadm init.
+		` + cmdutil.AlphaDisclaimer)
+
+	masterPreflightExample = normalizer.Examples(`
+		# Run master pre-flight checks.
+		kubeadm alpha phase preflight master
+		`)
+
+	nodePreflightLongDesc = normalizer.LongDesc(`
+		Run node pre-flight checks, functionally equivalent to what implemented by kubeadm join.
+		` + cmdutil.AlphaDisclaimer)
+
+	nodePreflightExample = normalizer.Examples(`
+		# Run node pre-flight checks.
+		kubeadm alpha phase preflight node
+	`)
+)
+
 // NewCmdPreFlight calls cobra.Command for preflight checks
 func NewCmdPreFlight() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "preflight",
 		Short: "Run pre-flight checks",
-		RunE:  cmdutil.SubCmdRunE("preflight"),
+		Long:  cmdutil.MacroCommandLongDescription,
```

Comment:
does this still `exit 1` when run w/o args?

Reply:
No, it `exit 0`, similar to `kubectl` invoked without any subcommand